gradle/publish-jar.gradle

```diff
@@ -17,7 +17,7 @@ bintray {
     publish = true
     pkg {
         repo = 'releases'
-        name = rootProject.name
+        name = 'conjure-java'
```

Comment:
would it be cleaner to update the root project name in settings.gradle?

Reply:
We can't because of the [consistent-versions limitation](https://github.com/palantir/gradle-consistent-versions#known-limitation-root-project-must-have-a-unique-name) :(